### ISSUE TITLE
Suggestion - Auto suggest number of headlands 

### ISSUE THREAD

**Issue by coatsy35** [Courseplay/CourseGenerator#12]
Can/could the new generator auto suggest the number of headlands based on the total length of the vehicle combination? and also display the total length as a number so users know how long a combination is?

**Comment by pvaiko** [comment]
Yeah, that was brought up many times (and it really isn't the duty of the generator). I think the reason we never implemented it is the many factors going in to that calculation, beside the length (actually it is rather what we call the back marker distance) you have to know which turn maneuver is feasible there (3 point, U, or a bulb, etc.) based on the turn radius, the working width, the implements front and back and the target row you want to end up in after the turn, and the angle of headland and who knows what else.

If you have the magic formula or an idea for a simplified calculation, let us know :)

**Comment by coatsy35** [comment]
Hmm, good point. Perhaps just displaying the length of the combination would be good enough. Or even globally save the headland setting for specific implement/vehicle combinations so it didn’t have to be entered each time. Just thinking out loud, but I see your point not an easy one. Perhaps an in game settings list for different implements rather than having to do it external in the CP files, so radius of turn path could be modified in game and saved along with the number of default headlands for that implement, type of headland turn (fishtail, bulb etc…) could have initial defaults and then allow users to change and save or reset etc… but that’s going off topic from the course generator.

**Comment by Tensuko** [comment]
I would go with the save bet and read the lenght + turn radius * 1.2 or something. Even it would be maybe too many headlands, it will help beginners at 1st (just an Idea).
